manifest.md

```diff
@@ -139,7 +139,7 @@ Unlike the [Manifest List](#manifest-list), which contains information about a s
 
 - **`layers`** *array*
 
-    Each item in the array MUST be a [descriptor](descriptor.md).
+    This property MAY be empty, an image can have an empty layers array. Each item in the array MUST be a [descriptor](descriptor.md).
```

Comment:
I think this should be “This OPTIONAL property”.  And we should stick to the [one sentence per line][1] rule.

To make this change clear, I think we also want the validation fix in #308, `omitempty` in the Go property, and dropping `layers` from `required` in `schema/image-manifest-schema.json`.

[1]: https://github.com/opencontainers/image-spec/tree/v0.4.0#markdown-style